unfoldingWord/translationCore
# Word Alignment: Using Verses For % Complete

#### This pull request addresses:
This PR makes the progress now use verses to calculate percent complete for the word alignment tool rather than the number of word alignments complete.


#### How to test this pull request:
Try doing some word alignments and make sure the side bar of progress with each group only gets more full once the entire verse is done being aligned.
Make sure the overall progress (Found on the tools page, and home page) is correlating to the amount of verses completely aligned not just partially aligned (based on the number of individual alignments).
See story for more info #2873


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3243)
